ODIN_II/SRC/include/config_t.h

```diff
@@ -38,9 +38,6 @@ struct config_t {
     int split_hard_adder;
     //  Threshold from hard to soft logic
     int min_threshold_adder;
-    // defines if the first cin of an adder/subtractor is connected to a global gnd/vdd
-    // or generated using a dummy adder with both inputs set to gnd/vdd
-    bool adder_cin_global;
```

Comment:
don't remove this. 
I would specify that if there's is a dummy carry, replace with this one.